### DISCUSSION TITLE
Not ready, Please check your connection

### DISCUSSION THREAD

**Discussion by redOz92** [rustdesk/rustdesk#4519] [Q&A]
Not ready, Please check your connection

I can connect to 3/5 of my servers and 2 servers get the message "Not ready, Please check your connection".
Checked the ID Server and everything is the same as the other servers, re-installed RustDesk but still nothing. Very strange and I can't find a solution myself..

Thanks in advance.

**Comment by rustdesk** [comment]
#4671
